src/plugins/discover/public/components/discover_grid_flyout/discover_grid_flyout.tsx

```diff
@@ -113,7 +115,9 @@ export function DiscoverGridFlyout({
     <EuiPortal>
       <EuiFlyout
         onClose={onClose}
+        type="push"
         size="m"
+        pushMinBreakpoint="xl"
         data-test-subj="docTableDetailsFlyout"
         onKeyDown={onKeyDown}
         ownFocus={false}
```

Comment:
I think `ownFocus` should be true now, this would not apply for the push, but for the overlay style
```suggestion
        ownFocus={true}
```